__tests__/app.test.js

```diff
@@ -286,100 +286,120 @@ describe("app", () => {
+      const newVote = { inc_votes: 3 };
+      return request(app)
+        .patch("/api/articles/LOL")
+        .send(newVote)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Invalid ID");
+        });
+    });
 
-  it("should return 400: PATCH responds with an error msg when the req body inc_votes does not have a number value", () => {
-    const newVote = { inc_votes: "rebel" };
-    return request(app)
-      .patch("/api/articles/3")
-      .send(newVote)
-      .expect(400)
-      .then(({ body }) => {
-        expect(body.msg).toBe("Bad Request");
-      });
+    it("should return 400: PATCH responds with an error msg when the req body inc_votes does not have a number value", () => {
+      const newVote = { inc_votes: "rebel" };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(newVote)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad Request");
+        });
+    });
+    it("should return 200 : PATCH updates the votes key on the correct article object and ignores any extra properties", () => {
+      const newVote = { inc_votes: -20, somethingExtra: "ohlala" };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(newVote)
+        .expect(202)
+        .then((article) => {
+          expect(article.body).toMatchObject({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 80,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+    });
   });
-  it("should return 200 : PATCH updates the votes key on the correct article object and ignores any extra properties", () => {
-    const newVote = { inc_votes: -20, somethingExtra: "ohlala" };
-    return request(app)
-      .patch("/api/articles/1")
-      .send(newVote)
-      .expect(202)
-      .then((article) => {
-        expect(article.body).toMatchObject({
-          article_id: 1,
-          title: "Living in the shadow of a great man",
-          topic: "mitch",
-          author: "butter_bridge",
-          body: "I find this existence challenging",
-          created_at: "2020-07-09T20:11:00.000Z",
-          votes: 80,
-          article_img_url:
-            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+
+  describe("GET /api/users", () => {
+    it("should return 200 status : GET responds with an array of ALL user objects with username, name, avatar_url properties", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((usersData) => {
+          const users = usersData.body;
```

Comment:
Key of users on the body would be more suitable, has this been done on other endpoints?